### ISSUE TITLE
Support `attr` as a value

### ISSUE THREAD

**Issue by hjonasson** [csstree/stylelint-validator#47]
Hi there,

I am cleaning up our stylelint config and I came across what seems to be a missing feature in the stylelint validator. Namely, to write this

```CSS
content: attr(title);
```

I need to have an `ignoreValue` pattern, (which I write `attr.*` for reference). I can not see from the docs that using `attr`  is supported by the validator. Could you show me how I am wrong about that or point me in a direction to where I could add this support here?

Thanks

**Comment by lahmatiy** [comment]
Update `css-tree` to [2.2.0](https://github.com/csstree/csstree/releases/tag/v2.2.0), should work.

**Comment by hjonasson** [comment]
Awesome stuff. Thanks Roman!

**Comment by hjonasson** [comment]
I am just getting around to doing that now. Does `stylelint-csstree-validator` need to have that release created? The latest version is from 2021 and `2.0.0` is the latest version on [npm](https://www.npmjs.com/package/stylelint-csstree-validator)

For reference, my dependency is 

```JSON
    "stylelint-csstree-validator": "^2.0.0",

```

**Comment by lahmatiy** [comment]
Fixed in [2.1.0](https://github.com/csstree/stylelint-validator/releases/tag/v2.1.0)

**Comment by hjonasson** [comment]
Thanks a lot Roman!
